app/src/main/storage.ts

```diff
@@ -86,20 +86,26 @@ export class Storage {
       "SecureDrop",
       "files",
     );
-    this.tmp = new PathBuilder(os.tmpdir() + "/");
+    this.tmp = new PathBuilder(fs.realpathSync(os.tmpdir()) + "/");
```

Comment:
Let me just add a comment explaining that this is a symlink on macOS so we know why it's done in the future.